src/Momento.Sdk/Responses/CreateCacheResponse.cs

```diff
@@ -1,8 +1,23 @@
 ﻿namespace Momento.Sdk.Responses;
 
-public class CreateCacheResponse
-{
-    public CreateCacheResponse()
+using Momento.Sdk.Exceptions;
+
+public class CreateCacheResponse {
```

Comment:
we can make this abstract now so that it can't be instantiated directly.  not a huge deal but might as well.

Reply:
Fixed in a6184ba5